esg_init.py

```diff
@@ -238,84 +286,32 @@ def init():
     #--------------------------------
     # External programs' versions
     #--------------------------------
-    openssl_version = "0.9.8r"
-    openssl_min_version = "0.9.8e"
-    openssl_max_version = "0.9.9z"
     java_version = "1.8.0_162"
     java_min_version = "1.8.0_162"
     ant_version = "1.9.1"
     ant_min_version = "1.9.1"
-    postgress_version = "8.4.20"
-    postgress_min_version = "8.4.20"
-    tomcat_version = "8.5.9"
-    tomcat_min_version = "8.5.9"
-    tds_version = "5.0.2"
-    tds_min_version = "5.0.2"
     sqlalchemy_version = "0.7.10"
     sqlalchemy_min_version = "0.7.10"
     python_version = "2.7"
 
     #--------------------------------
     # Script vars (~external)
     #--------------------------------
-    openssl_install_dir = os.path.join(install_prefix, "openssl")
-    postgress_install_dir = os.path.join("/","var", "lib", "pgsql")
-    postgress_bin_dir = os.path.join("/","usr", "bin")
-    postgress_lib_dir = os.path.join("/","usr", "lib64", "pgsql")
-    postgress_user = "dbsuper"
-    pg_sys_acct_passwd = "changeme"
-    pub_secret = ""
-    publisher_db_user_passwd = ""
-    try:
-        with open(pub_secret_file, 'rb') as filedata:
-            pub_secret = filedata.read().strip()
-        publisher_db_user_passwd = pub_secret
-    except IOError, error:
-        logger.debug(error)
-
-    postgress_host = "localhost"
-    postgress_port = "5432"
     # Double Check HERE
-    cdat_home = os.path.join(install_prefix, "uvcdat", "2.2.0")
     java_opts = ""
     java_install_dir = os.path.join(install_prefix, "java")
     ant_install_dir = os.path.join(install_prefix, "ant")
-    tomcat_install_dir = os.path.join(install_prefix, "tomcat")
-    tomcat_conf_dir = os.path.join(esg_config_dir, "tomcat")
-    tomcat_opts = ""
-    tomcat_user = "tomcat"
-    tomcat_group = tomcat_user
     globus_location = os.path.join(install_prefix, "globus")
     mail_smtp_host = "smtp."+socket.getfqdn().split('.', 1)[1]
     mail_admin_address = ""
-    publisher_home = ""
-    publisher_config = ""
-    ESGINI = ""
-    try:
-        os.environ["ESGINI"]
-    except KeyError:
-        publisher_home = os.path.join(esg_config_dir, "esgcet")
-        publisher_config = "esg.ini"
-        ESGINI = os.path.join(publisher_home, publisher_config)
 
     ############################################
     ####  DO NOT EDIT BELOW THIS POINT!!!!! ####
     ############################################
     os.environ["GIT_SSL_NO_VERIFY"] = "1"
```

Comment:
Theses os.environ can be deprecated. Or they need to be refactored to be written out to a separate file